doc/pylint.rst

```diff
@@ -1,35 +1,95 @@
-Or, you can globally suppress certain messages by editing the :file:`.pylintrc` configuration file in your user home directory.
+To go directly to a line in the :doc:`editor` highlighted by a failed check, just click its name.
+
+.. image:: images/pylint/code_analysis_editor.gif
+   :alt: Spyder Pylint pane, showing running analysis and clicking failed check
+
+|
+
+You can manually enter the Python module or package path you'd like it to check in the path entry box in the pane's toolbar.
+The analyzer works with both individual scripts and whole Python packages (directories containing an :file:`__init__.py` file).
+
+.. image:: images/pylint/code_analysis_file.gif
+   :alt: Spyder Pylint pane, showing running analysis browsing file
+
+|
+
+Cancel analyzing a file with the :guilabel:`Stop` button, and if analysis fails, click the :guilabel:`Output` button to find out why. 
+
+.. image:: images/pylint/code_analysis_stop.gif
+   :alt: Spyder Pylint pane, showing stopping analysis
+
+|
+
+You can click the dropdown or press the :kbd:`Down Arrow` key in the filename field to view results of previous analyses.
+
+
+
+===================
+Contex menu options
+===================
+The number of recent runs Spyder should remember can be customized in the :guilabel:`History` dialog, available from the :guilabel:`Code Analysis` context menu.
+
+.. image:: images/pylint/code_analysis_history.gif
+   :alt: Spyder Pylint pane, customizing history dialog
+
+|
+
+You can also expand or collapse one or all the sections in the pane by using the corresopnding options in the context menu.
+
+.. image:: images/pylint/code_analysis_expand_collapse.gif
+   :alt: Spyder Pylint pane, expanding and collapsing sections
+
+|
+
+
+
+================
+Advanced options
+================
+
+You can turn certain messages off at the line, block or file/module level by adding a ``# pylint: disable=<list of message names>`` comment at the respective `level`_, (LINK) where the ``<...>`` portion should be replaced with a comma-separated list (or single value) without the ``<``/``>`` of Pylint message names (*e.g.* ``multiple-statements``, or ``fixme, line-too-long``; you can also use `error codes`_ like ``C0321`` although names are preferred for clarity).  
+
+.. _Pylint message names: http://pylint.pycqa.org/en/latest/technical_reference/features.html#pylint-checkers-options-and-switches
+
+.. image:: images/pylint/code_analysis_pylint_disable.gif
+   :alt: Spyder Pylint pane, showing disabling a message locally
+
+|
+
+Or, you can globally suppress specific messages and adjust other Pylint settings by editing the :file:`.pylintrc` configuration file in your user folder.
+If it doesn't exist, you can generate it by running `pylint --generate-rcfile > .pylintrc` in your home directory from Anaconda Prompt (on Windows) or your terminal (macOS/Linux).
```

Comment:
```suggestion
If it doesn't exist, you can generate it by running `pylint --generate-rcfile > .pylintrc` in your Users directory from the Anaconda Prompt (on Windows) or your terminal (on macOS/Linux).
```

* Probably Windows users don't know what their `home` directory is. Besides, `Users` is the same for both Windows and macOS, and almost all Linux users know that it refers to their `home` dir.
* I think `the` is missing here? (Not really sure).

Reply:
```suggestion
If it doesn't exist, you can generate it by running `pylint --generate-rcfile > .pylintrc` in your user directory, from Anaconda Prompt (on Windows) or your terminal (macOS/Linux).
```

I guess, I just wanted to avoid repetition. The correct term should be `user` directory; on Windows and macOS the `Users` directory is the directory that stores the home directories.